vm/vmExpr.go

```diff
@@ -756,7 +756,23 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 		}
 
 		for i := 0; i < listExpr.Len(); i++ {
-			if reflect.DeepEqual(listExpr.Index(i).Interface(), itemExpr.Interface()) {
+			item := itemExpr
+			list := listExpr.Index(i)
+
+			if item.Kind() == reflect.Interface || item.Kind() == reflect.Ptr {
+				item = item.Elem()
+			}
+			if list.Kind() == reflect.Interface || list.Kind() == reflect.Ptr {
+				list = list.Elem()
+			}
+
+			if isNum(item) && isNum(list) {
```

Comment:
Just questino. Why only number?

Reply:
if not this, than 
int(1) != int64(1) when use input, etc...

Reply:
ok~